include/libdnf-cli/smartcols.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef LIBDNF_CLI_UTILS_SMARTCOLS
+#define LIBDNF_CLI_UTILS_SMARTCOLS
```

Comment:
move the file to  include/libdnf-cli/output/output.hpp
change to LIBDNF_CLI_OUTPUT_OUTPUT_HPP